.rpmignore

```diff
@@ -3,28 +3,19 @@
 # but should not be included in the current release
 #
 
-centos7/daos-client-tests-openmpi*.rpm
-centos7/daos-firmware*.rpm
-centos7/daos-mofed*.rpm
-centos7/daos-serialize*.rpm
-centos7/daos-server-tests-openmpi*.rpm
-centos7/daos-tests-internal*.rpm
+centos7/daos-*.rpm
 centos7/ucx*.rpm
```

Comment:
Are any of these needed/wanted in the 2.4 copy of this file given that CentOS 7 is not supported or built on this release?

Reply:
Totally ignored by the scripts for versions we do not ship el-7 for.

Reply:
Understood.  Was suggesting as a tidy-up operation for dead code.

Reply:
sounds good to me, i reduced the centos7 entries from 7 to 2 in the same spirit, but happy to reduce them to zero ;-)